stage0/stop.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2014 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//+build linux
+
+package stage0
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"syscall"
+
+	"github.com/coreos/rkt/common"
+)
+
+func StopPod(dir string) error {
+	s1rootfs := common.Stage1RootfsPath(dir)
+
+	debug("Pivoting to filesystem %s", dir)
```

Comment:
I know it comes from `stage0/run.go` but since this code does not use `pivot_root(2)`, can we remove the word "Pivoting"?